pq_logic/keys/sig_keys.py

```diff
@@ -587,15 +607,25 @@ def from_private_bytes(cls, data: bytes, name: str) -> "SLHDSAPrivateKey":
 
         :param data: The byte string to create the private key from.
         :param name: The name of the signature algorithm.
+        :return: The loaded SLHDSAPrivateKey.
+        :raises ValueError: If the key name is not supported.
+        :raises InvalidKeyData: If the key data is invalid.
         """
         obj = cls(alg_name=name)
         _n = obj._slh_class.n
 
         if len(data) == 3 * _n:
             s_key = cls.from_seed(alg_name=name, seed=data)
 
+        elif len(data) == 7 * _n:
+            seed = data[: 3 * _n]
+            private_key = data[3 * _n : 7 * _n]
```

Comment:
Here it would help to know where the numbers 3 and 7 come from; perhaps a reference or a one or two lines to explain it?